src/config/index.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable global-require */
+import * as _ from 'lodash';
+
+export interface Config {
```

Comment:
I just think that the config can be moved to a separate npm package in the future. 
Just an idea